src/contributor-guide/dod/tco-analysis.md

```diff
@@ -0,0 +1,114 @@
+| Events triggered by a component        | Event argument removed                                                                                                   | MAJOR |
+|                                        | Event removed                                                                                                            | MAJOR |
+|                                        | Event added                                                                                                              | MINOR |
+| PHP Interface (marked with @api)       | Method parameter default value removed                                                                                   | MAJOR |
+|                                        | Method parameter default value changed                                                                                   | MAJOR |
+|                                        | Method parameter default value added                                                                                     | MINOR |
+|                                        |                                                                                                                          |       |
+| PHP Class (marked with @api)           | Method argument or return value format changed (Example: before - [1, 2, 3], after - ['items' => [1,2,3], 'count' => 3]) | MAJOR |
+|                                        | Trait removed                                                                                                            | MAJOR |
+| Message queue                          | Topic removed                                                                                                            | MAJOR |
+|                                        | Consumer removed                                                                                                         | MINOR |
+|                                        | New topic published                                                                                                      | MINOR |
+| Layout handles declared by modules     | New layout page handle                                                                                                   | MINOR |
+|                                        | New container/block added to handle                                                                                      | MINOR |
+|                                        | Removed layout handle                                                                                                    | MAJOR |
+| Static view files                      | JS/CSS/LESS file removed/moved                                                                                           | MAJOR |
+
+### Potential number of affected customisations
+
+In order to identify the volume of the impact the usages of the changed code should be detected in the available code/extensions/customisations.
```

Comment:
```suggestion
In order to identify the volume of the impact the usages of the changed code should be detected in the available code/extensions/customizations.
```

Reply:
It's a tricky one.

In the U.S. and Canada, it’s customize, customized, customizing, customization, customizable, etc. Outside North America, it’s customise, customised, customising, customisation, customisable, etc.

Considering that I am writing this documentation in Ireland - "s" i used :D

Reply:
:) Okey, did know about that :)